src/app/components/CurrencyAsset/index.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ *
+ * CurrencyAssets
+ *
+ */
+import React from 'react';
+import styled from 'styled-components/macro';
+
+import { Asset } from 'types/asset';
+
+import { AssetsDictionary } from '../../../utils/dictionaries/assets-dictionary';
+
+interface CurrencyProps {
+  assetImg?: any;
+  assetSymbol: Asset;
+}
+const symbolMap = {
+  [Asset.RBTC]: 'rBTC',
+  [Asset.USDT]: 'rUSDT',
+};
+// function renderCustomSymbol(currency: string) {
+//   return (
+//     <div className="d-flex flex-row">
+//       <Small>{currency.charAt(0)}</Small>
+//       {currency.slice(1)}
+//     </div>
+//   );
+// }
+function getAssetSymbol(asset: Asset) {
+  if (symbolMap.hasOwnProperty(asset)) {
+    return symbolMap[asset];
```

Comment:
Have a wrapper for symbol renderer, children <small> node must be styled to be 75% height of normal text and have 50% opacity.
styled-components can be used for this.

```suggestion
    return <span className="symbol">{symbolMap[asset]}</span>;
```